packages/lab/stories/input-next/input-next.stories.tsx

```diff
@@ -23,6 +24,23 @@ export const Secondary: ComponentStory<typeof InputNext> = (args) => {
   );
 };
 
+export const Controlled: ComponentStory<typeof InputNext> = (args) => {
+  const [value, setValue] = useState("Value");
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    setValue(value);
+  };
+
+  return (
+    <InputNext
+      {...args}
+      value={value}
+      onChange={handleChange}
```

Comment:
`onChange={(event) => { setValue(event.target.value); }}`
@alycrys this would cause typecheck errors for us due to us using the default onChange method - is this going to be a potential problem for users too? (can be handled with the current implementation)

Reply:
I believe this is the correct approach